### PR TITLE
remote: add description for verify option

### PR DESCRIPTION
**Pull request by pared**
Disregard the recommendations below if you use **Edit on GitHub** button to improve the docs in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This enables GitHub to link the PR to the corresponding bug and close it automatically when PR is merged.

Thank you for the contribution - we'll try to review and merge it as soon as possible. 🙏

----------
related to https://github.com/iterative/dvc/pull/3200

### CODE REVIEW

File: public/static/docs/command-reference/remote/modify.md [hunk 1]
```diff
@@ -59,6 +59,18 @@ manual editing could be used to change the configuration.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
+## Avialable settings for all remotes
+
+The following options are avialable for all remotes:
```
Comment (jorgeorpinel) [inline public/static/docs/command-reference/remote/modify.md]:
```suggestion
The following options are available for all remote types:
```

File: public/static/docs/command-reference/remote/modify.md [hunk 2]
```diff
@@ -59,6 +59,18 @@ manual editing could be used to change the configuration.
...
+
+- `verify` - upon downloading cache files (`dvc pull`, `dvc fetch`) DVC will
+  recalculate downloaded file checksum to make sure that it is not corrupted or
```
Comment (jorgeorpinel) [inline public/static/docs/command-reference/remote/modify.md]:
```suggestion
  recalculate downloaded file checksums to make sure that these are not corrupted or
```

File: public/static/docs/command-reference/remote/modify.md [hunk 3]
```diff
@@ -59,6 +59,18 @@ manual editing could be used to change the configuration.
...
+  modified during download. It may slow down mentioned commands.
```
Comment (jorgeorpinel) [inline public/static/docs/command-reference/remote/modify.md]:
```suggestion
  modified during download. It may slow down the aforementioned commands.
```

**Comment by jorgeorpinel** [comment]
@pared also, is there a PR in the core repo that implements this? Would be nice to put that in the description of this PR. (The template text you can leave or just remove after reading it.)

Another hint: in the future in order to make docs process a little faster please use a branch in this repo directly, as opposed to your fork (easier for us to checkout and commit to). Thanks

**Comment by shcheklein** [comment]
@jorgeorpinel please don't hesitate to apply small changes right away and let's just confirm with @pared the latest version that is ready to merge + if we have any questions to clarify (better ask in the engineering team or Discord then)

**Comment by jorgeorpinel** [comment]
I don't think I can on this fork:

image

File: public/static/docs/command-reference/remote/modify.md [hunk 4]
```diff
@@ -59,6 +59,18 @@ manual editing could be used to change the configuration.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
+## Avialable settings for all remotes
```
Comment (jorgeorpinel) [inline public/static/docs/command-reference/remote/modify.md]:
```suggestion
## Available settings for all remotes
```
p.s. consider using a spell checker in your IDE.

Comment (jorgeorpinel) [reply public/static/docs/command-reference/remote/modify.md]: I'll fix this.

**Comment by shcheklein** [comment]
@pared @jorgeorpinel looks good to me, but I think it worth mentioning what remote types have this setting on/off by default.

@jorgeorpinel might worth mentioning that we take this checksums from the DVC-files.

**Comment by jorgeorpinel** [comment]
> worth mentioning what remote types have this setting on/off by default

This I don't know. I asked in the #engineering channel and Ruslan mentioned only on Gdrive. Added corresponding notes in 1185efcc.

He also mentioned its still a WIP PR on the core repo. So we shouldn't merge this just yet. (Another reason to try including the link to the core PR on these docs PRs.)

**Comment by jorgeorpinel** [comment]
> mentioning that we take this checksums from the DVC-files.

Done as well @shcheklein

**Comment by pared** [comment]
Ok, thanks @jorgeorpinel for taking time to configure your repo, Ill remember to push to origin branch, didn't know that sanity checks are triggered only on branch PR's.
Added related issue on dvc side.

**Comment by pared** [comment]
Original change has been merger @shcheklein @jorgeorpinel

**Comment by shcheklein** [comment]
@pared 

> didn't know that sanity checks are triggered only on branch PR's.

I believe it runs CI checks on a fork as well, but does not deploy on Heroku automatically to review

**Comment by shcheklein** [comment]
@jorgeorpinel merging this?

**Comment by jorgeorpinel** [comment]
> didn't know that sanity checks are triggered only on branch PR's

Ah yeah the CI checks run. I meant that a review app is deployed on Heroku so we can do manual sanity checks visually on a rendered website. But yeah thanks and np @pared 

> Original change has been merger

OK, merging here as well 😬

**Comment by shcheklein** [comment]
@jorgeorpinel looks like build fails after this merge

**Pull request merged**